NuGet/Cotecna.Domain.Core/Cotecna.Domain.Core.csproj

```diff
@@ -10,12 +10,12 @@
 		<Authors>Cotecna Inspection</Authors>
 		<Company>Cotecna Inspection</Company>
 		<RestoreProjectStyle>PackageReference</RestoreProjectStyle>
-		<Version>1.4.0</Version>
-		<FileVersion>1.4.0</FileVersion>
-		<AssemblyVersion>1.4.0</AssemblyVersion>
+		<Version>1.4.1</Version>
+		<FileVersion>1.4.1</FileVersion>
+		<AssemblyVersion>1.4.1</AssemblyVersion>
 		<PackageLicenseFile>LICENSE</PackageLicenseFile>
 		<PackageProjectUrl>https://github.com/Cotecna-Inspection/Domain.Core</PackageProjectUrl>
-		<PackageReleaseNotes>https://github.com/Cotecna-Inspection/Domain.Core/tree/main/NuGet/Cotecna.Domain.Core/blob/main/changelog.md</PackageReleaseNotes>
+		<PackageReleaseNotes>https://github.com/Cotecna-Inspection/Domain.Core/blob/main/NuGet/Cotecna.Domain.Core.Configuration/changelog.md</PackageReleaseNotes>
```

Comment:
now it's pointint to tha change of configuration

Reply:
Should be


https://github.com/Cotecna-Inspection/Domain.Core/blob/main/NuGet/Cotecna.Domain.Core/changelog.md

Reply:
Done!